docs/md/workload.md

```diff
@@ -0,0 +1,47 @@
+# Workloads
+
+## What is it?
+Within the context of the workflow launcher, a workload is a discrete body of work, which takes data from a source, pushes it into a workflow executor for analysis, and then delivers the results of the analysis to an output location (also known as a sink).
+
+## Workload Components
+### Source
+### Executor
+### Sink
+
+## Example workload 
+(The specific values below are from the COVID-19 Surveillance in Terra project. Workloads for other projects may have different fields available in the source, executor and sink).
+
+```
+{
+    "watchers": [],
+    "labels": [
+        "hornet:test"
+    ],
+    "project": "wfl-dev/CDC_Viral_Sequencing",
```

Comment:
It would be good to explain what these are for

Reply:
I tried to address some of this, but didn't know if I should fill in values for source, sink, and exec since other tickets cover this.